packages/mobile/screens/SignIn.tsx

```diff
@@ -0,0 +1,203 @@
+import { useMutation } from "@apollo/react-hooks";
+import { Field, Formik } from 'formik';
+import gql from "graphql-tag";
+import * as React from "react";
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+import { TextInput } from "react-native-paper";
+import styled from 'styled-components/native';
+import * as Yup from "yup";
+import LOG_IN_MUTATION from '../local_core/mutations/LOG_IN_MUTATION';
+
+const LoginSchema = Yup.object().shape({
+    email: Yup.string()
+        .email("Please enter a valid email")
+        .required("Please enter an email"),
+    password: Yup.string().required("Required")
+});
+
+
+export default function SignUp({ navigation }: { navigation: any }) {
```

Comment:
Please define the function using arrow functions and then use export default with it, it improves readability

Reply:
improve readibility?????

Reply:
Yes, what I mean with that is that it's easier to understand the code. Also it seems that the "SignIn" screen is exporting a "SignUp" function?

Reply:
i already changed name